website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-page_title: Policy Checks - API Docs - HCP Terraform
+page_title: Policy checks API for HCP Terraform
 description: >-
-  Use the `/policy-checks` endpoint to manage the Sentinel checks performed on a Terraform run. List, show and override policy checks using the HTTP API.
+  Use the `/policy-checks` endpoint to manage and override the Sentinel policy checks that HCP Terraform performs during a run.
```

Comment:
```suggestion
  Use the HCP Terraform API's `/policy-checks` endpoint to manage and override the Sentinel policy checks that HCP Terraform performs during a run.
```

Looks like a couple of these don't include this bit, wasn't sure if that was for a particular reason so feel free to reject if it was